### ISSUE TITLE
Weekly Sales: Keyboard Navigation

### ISSUE THREAD

**Issue by KTottenDC** [DouglasCountyCO/Accessibility-PowerBI-Reports-Public#24]
Fail Tab Order

Order Needs "Data Story" navigation for tab order.

![image](https://github.com/DouglasCountyCO/Accessibility-PowerBI-Reports-Public/assets/127148669/9a87351f-e427-44d4-825a-4b7ddb05947e)

No traps detected.
Enter allowed into the visual component.
Date And Time Data was Last Refreshed worked.
Public Trustee Resources - all links accessible.

**Comment by KTottenDC** [comment]
Tab Order needs to be addressed.
